packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -160,6 +191,14 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     return this?.state?.isStreaming;
   }
 
+  get shouldDisplayFeedback() {
+    return this.isVisible && !this.isStreaming;
+  }
+
+  get isVisible() {
+    return this?.state?.isVisible;
```

Comment:
I don't like this as this will return `undefined` if the state .isVisible is missing, which means the genQA will be hidden.

Reply:
Practically this will never be undefined, as the state in Headless is always defined with the `isVisible` property that has an initial value of `true`, I can remove the `?` to remove the ambiguity.

Reply:
I would say yes to that or default to specific true or false. Don't let it run with a potentially ambiguous "undefined"